AdvancedTask/Pages/ManageRequestPage.cs

```diff
@@ -1,12 +1,170 @@
+
+            var messages = new List<string>();
+            foreach (var element in messageElements)
+            {
+                messages.Add(element.Text);
+
+            }
+            return messages.ToArray();
+        }
+        public void SortByRecipient()
+        {
+            //Identify category and click
+            recipient.Click();
+            Thread.Sleep(500);
+        }
+
+        public string[] GetRecipients()
+        {
+            var recipientElements = driver.FindElements(By.XPath("//*[@id=\"sent-request-section\"]/div[2]/div[1]/table/tbody/tr/td[4]"));
+
+            var recipients = new List<string>();
+            foreach (var element in recipientElements)
+            {
+                recipients.Add(element.Text);
+
+            }
+            return recipients.ToArray();
+        }
+        public void SortByStatus()
+        {
+            //Identify category and click
+            IWebElement status = driver.FindElement(By.XPath("//*[@id=\"sent-request-section\"]/div[2]/div[1]/table/thead/tr/th[5]"));
+            status.Click();
+            Thread.Sleep(500);
+        }
+
+        public string[] GetStatuses()
+        {
+            var statusElements = driver.FindElements(By.XPath("//*[@id=\"sent-request-section\"]/div[2]/div[1]/table/tbody/tr/td[5]"));
+
+            var statuses = new List<string>();
+            foreach (var element in statusElements)
+            {
+                statuses.Add(element.Text);
+
+            }
+            return statuses.ToArray();
+        }
+        public void SortByDate()
+        {
+            //Identify category and click
+            IWebElement dates = driver.FindElement(By.XPath("//*[@id=\"sent-request-section\"]/div[2]/div[1]/table/thead/tr/th[7]"));
+            dates.Click();
+            Thread.Sleep(500);
+        }
+
+        public string[] GetDates()
+        {
+            var datesElements = driver.FindElements(By.XPath("//*[@id=\"sent-request-section\"]/div[2]/div[1]/table/tbody/tr/td[7]"));
+            var dates = new List<string>();
+            foreach (var element in datesElements)
+            {
+                dates.Add(element.Text);
+
+            }
+            return dates.ToArray();
+        }
+
+        public void WithdrawSentRequests()
+        {
+            //Identify withdraw button and click
+            Wait.WaitToBeClickable(driver, "XPath", "//*[@id=\"sent-request-section\"]/div[2]/div[1]/table/tbody/tr[1]/td[8]/button", 15);
+            withdrawActionButton.Click();
+           
+        }
+
+        public string GetPopUpMessage()
+        {
+            Wait.WaitToBeVisible(driver, "CssSelector", ".ns-box-inner", 15);
+            return popUpmMessage.Text;
+        }
+        private IWebElement manageRequest => driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[1]/div/div[1]"));
```

Comment:
something small but best practice to move all your instance variable before method and constructor as its easier to read code